### DISCUSSION TITLE
build up with docker but can't run exmaple normally

### DISCUSSION THREAD

**Discussion by nieyan** [StanfordVL/BEHAVIOR-1K#1041] [Q&A] [answered]
```python -m omnigibson.examples.objects.load_object_selector```
Choose a object model (enter a number from 1 to 92): 75
[INFO] [omnigibson.simulator] ----- Starting OmniGibson. This will take 10-30 seconds... -----
Starting kit application with the following args:  ['/isaac-sim/exts/omni.isaac.kit/omni/isaac/kit/simulation_app.py', '/isaac-sim/apps/omnigibson_4_1_0.kit', '--/app/tokens/exe-path=/isaac-sim/kit', '--/persistent/app/viewport/displayOptions=3094', '--/rtx/materialDb/syncLoads=True', '--/rtx/hydra/materialSyncLoads=True', '--/omni.kit.plugin/syncUsdLoads=True', '--/app/renderer/resolution/width=1280', '--/app/renderer/resolution/height=720', '--/app/window/width=1440', '--/app/window/height=900', '--/renderer/multiGpu/enabled=False', '--/app/fastShutdown=True', '--ext-folder', '/isaac-sim/exts', '--ext-folder', '/isaac-sim/apps', '--/physics/cudaDevice=0', '--portable', '--allow-root']
Passing the following args to the base kit application:  []
[Info] [carb] Logging to file: /isaac-sim/kit/logs/Kit/Isaac-Sim/4.1/kit_20241127_123205.log
2024-11-27 12:32:05 [0ms] [Warning] [omni.kit.app.plugin] No crash reporter present, dumps uploading isn't available.
[3.872s] [ext: omni.kit.async_engine-0.0.0] startup
[DEBUG] [asyncio] Using selector: EpollSelector
[6.756s] [ext: omni.stats-1.0.1] startup
[6.768s] [ext: omni.client-1.1.0] startup
[7.258s] [ext: omni.datastore-0.0.0] startup
[7.276s] [ext: omni.blobkey-1.1.0] startup
[7.291s] [ext: omni.ujitso.default-1.0.0] startup
[7.319s] [ext: omni.hsscclient-0.0.0] startup
[7.338s] [ext: omni.rtx.shadercache.vulkan-1.0.0] startup
[7.341s] [ext: omni.assets.plugins-0.0.0] startup
[7.470s] [ext: omni.gpu_foundation-0.0.0] startup
[8.274s] [ext: carb.windowing.plugins-1.0.0] startup
2024-11-27 12:32:21 [15,684ms] [Warning] [carb.windowing-glfw.gamepad] Joystick with unknown remapping detected (will be ignored):  QEMU QEMU USB Tablet [03000000270600000100000001000000]
[15.896s] [ext: omni.kit.renderer.init-0.0.0] startup
X Error of failed request:  GLXBadFBConfig
  Major opcode of failed request:  150 (GLX)
  Minor opcode of failed request:  0 ()
  Serial number of failed request:  179
  Current serial number in output stream:  179

==================================================
omnigibson-src# nvidia-smi
Wed Nov 27 12:40:11 2024       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.183.01             Driver Version: 535.183.01   CUDA Version: 12.2     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 3090        Off | 00000000:01:00.0 Off |                  N/A |
| 61%   57C    P2             193W / 350W |  12547MiB / 24576MiB |     32%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   1  NVIDIA GeForce RTX 3090        Off | 00000000:02:00.0 Off |                  N/A |
| 57%   54C    P2             161W / 350W |  10920MiB / 24576MiB |    100%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   2  NVIDIA GeForce RTX 3090        Off | 00000000:03:00.0 Off |                  N/A |
| 32%   33C    P8              12W / 350W |      3MiB / 24576MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   3  NVIDIA GeForce RTX 3090        Off | 00000000:04:00.0 Off |                  N/A |
| 31%   33C    P8              14W / 350W |      3MiB / 24576MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
+---------------------------------------------------------------------------------------+

**Comment by hang-yin** [comment]
@cgokmen have you seen this before?

**Reply by nieyan** [reply, marked as answer]
It's because I run docker in a ubuntu server with GUI but without monitor. I try with a normal ubuntu laptop is good. Thanks for your attention !
